.circleci/config.yml

```diff
@@ -41,21 +47,19 @@ commands:
     description: "Add changes to SVN"
     steps:
       - run:
-          command: if [[ ! -z $(svn st | grep ^\!) ]]; then svn st | grep ^! | awk '{print " --force "$2}' | xargs -0r svn rm; fi
-      - run: svn add --force .
+          command: |
+            SVN_DIRECTORIES="trunk tags"
+            svn stat $SVN_DIRECTORIES | { grep -E '^\?' || true; } | awk '{print $2}' | xargs -r svn add
+            svn stat $SVN_DIRECTORIES | { grep -E '^\!' || true; } | awk '{print $2}' | xargs -r svn rm
+            echo "Here is the svn stat about to be checked in:"
+            svn stat
```

Comment:
Here's the result of running this locally:

```sh
echo "Here is the svn stat about to be checked in:"
svn stat

D         tags/3.1.0/svgxuse.js
D         trunk/svgxuse.js
Here is the svn stat about to be checked in:
A  +    tags/3.1.0
M  +    tags/3.1.0/README.md
M  +    tags/3.1.0/composer.json
M  +    tags/3.1.0/package.json
M  +    tags/3.1.0/readme.txt
M  +    tags/3.1.0/simple-social-icons.php
D  +    tags/3.1.0/svgxuse.js
 M      trunk
M       trunk/README.md
M       trunk/composer.json
M       trunk/package.json
M       trunk/readme.txt
M       trunk/simple-social-icons.php
D       trunk/svgxuse.js
```

Reply:
That looks good, when checking against the tag's [diff](https://github.com/studiopress/simple-social-icons/compare/3.0.2...3.1.0)

Reply:
Actually, `tags/3.1.0` is missing `css/` and `icons/`

Reply:
With `svn stat --verbose`, it looks like `tags/3.1.0/css/` and `tags/3.1.0/icons` are included:

```
D         tags/3.1.0/svgxuse.js
D         trunk/svgxuse.js
Here is the svn stat about to be checked in:
           2711177  2284431 studiopress  .
           2711177  2284431 studiopress  tags
A  +             -  2282620 studiopress  tags/3.1.0
M  +             -  2092385 studiopress  tags/3.1.0/README.md
M  +             -  2282620 studiopress  tags/3.1.0/composer.json
   +             -  1548617 nathanrice   tags/3.1.0/css
   +             -  1548617 nathanrice   tags/3.1.0/css/style.css
   +             -  1954571 studiopress  tags/3.1.0/icons
   +             -  1546675 nathanrice   tags/3.1.0/icons/LICENSE.txt
   +             -  1954571 studiopress  tags/3.1.0/icons/SVG
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/behance.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/bloglovin.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/dribbble.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/email.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/facebook.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/flickr.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/github.svg
   +             -  1549615 nathanrice   tags/3.1.0/icons/SVG/gplus.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/instagram.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/linkedin.svg
   +             -  1954571 studiopress  tags/3.1.0/icons/SVG/medium.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/periscope.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/phone.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/pinterest.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/rss.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/snapchat.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/stumbleupon.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/tumblr.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/twitter.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/vimeo.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/xing.svg
   +             -  1546675 nathanrice   tags/3.1.0/icons/SVG/youtube.svg
   +             -  2092385 studiopress  tags/3.1.0/package-lock.json
M  +             -  2282620 studiopress  tags/3.1.0/package.json
M  +             -  2282620 studiopress  tags/3.1.0/readme.txt
M  +             -  2282620 studiopress  tags/3.1.0/simple-social-icons.php
D  +             -       ?   ?           tags/3.1.0/svgxuse.js
   +             -  1954571 studiopress  tags/3.1.0/symbol-defs.svg
 M         2711177  2282620 studiopress  trunk
M          2711177  2092385 studiopress  trunk/README.md
M          2711177  2282620 studiopress  trunk/composer.json
           2711177  1548617 nathanrice   trunk/css
           2711177  1548617 nathanrice   trunk/css/style.css
           2711177  1954571 studiopress  trunk/icons
           2711177  1546675 nathanrice   trunk/icons/LICENSE.txt
           2711177  1954571 studiopress  trunk/icons/SVG
           2711177  1546675 nathanrice   trunk/icons/SVG/behance.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/bloglovin.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/dribbble.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/email.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/facebook.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/flickr.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/github.svg
           2711177  1549615 nathanrice   trunk/icons/SVG/gplus.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/instagram.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/linkedin.svg
           2711177  1954571 studiopress  trunk/icons/SVG/medium.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/periscope.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/phone.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/pinterest.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/rss.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/snapchat.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/stumbleupon.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/tumblr.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/twitter.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/vimeo.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/xing.svg
           2711177  1546675 nathanrice   trunk/icons/SVG/youtube.svg
           2711177  2092385 studiopress  trunk/package-lock.json
M          2711177  2282620 studiopress  trunk/package.json
M          2711177  2282620 studiopress  trunk/readme.txt
M          2711177  2282620 studiopress  trunk/simple-social-icons.php
D          2711177  1546675 nathanrice   trunk/svgxuse.js
           2711177  1954571 studiopress  trunk/symbol-defs.svg
```